expr/bitwise_test.go

```diff
@@ -0,0 +1,55 @@
+package expr
+
+import (
+	"encoding/binary"
+	"reflect"
+	"testing"
+
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+func TestBitwise(t *testing.T) {
```

Comment:
Can you mark this test as parallelizable by calling `t.Parallel()` as first statement please?

Reply:
done